CHANGELOG.rst

```diff
@@ -4,24 +4,29 @@ Changelog
 in development
 --------------
 
-* Move babel-eslint dependency to devDependencies (bug fix)
-* Small refactor and more tests (for ``scripts/stackstorm.js``) (improvement)
-* Refactor chat providers into their own modules (improvement)
-* Modernize directory structure to be more consistent with other hubot plugins (improvement)
-* Split out the functionality of ``src/stackstorm.js`` into ``stackstorm_api.js`` and refactor it be a JS old style class with a wrapper (improvement)
+* Move babel-eslint dependency to devDependencies [PR #180] (bug fix)
+* Small refactor and more tests (for ``scripts/stackstorm.js``) [PR #185] (improvement)
+* Refactor chat providers into their own modules [PR #186] (improvement)
+* Modernize directory structure to be more consistent with other hubot plugins [PR #186, PR #191]
+  (improvement)
+* Split out the functionality of ``src/stackstorm.js`` into ``stackstorm_api.js`` and refactor it
```

Comment:
Wasn't #190 a fix?

Reply:
My thinking was: #187 was an improvement (refactoring), #190 was a fix introduced in that refactoring.

So between the last release and the next release the combined result of #187 and #190 amounts to just an improvement, not an improvement+bugfix.

Reply:
Ideally to consider every PR as isolated change that should have dedicated Changelog record, instead of assuming that one PR is an addition to another.